source/_includes/installation/container/alternative.md

```diff
@@ -137,3 +137,9 @@ That will tell Home Assistant where to look for our Z-Wave radio.
 device_tracker:
   - platform: bluetooth_tracker
 ```
+
+### Asustor NAS
+
+- Install the [docker-ce package](https://www.asustor.com/app_central/app_detail?id=754) in the App Center.
+- Connect to your NAS over SSH.
+From there you can follow the steps for [Platform Installation](https://www.home-assistant.io/installation/generic-x86-64#platform-installation) in the Container section of Generic x86-x64 since the Docker CLI commands are the same.
```

Comment:
Thanks for willing to add additional, alternative installation documentation.

However, we have limited the number of installations to match the largest use cases of our users in our documentation (we don't want to end up with potentially hundreds of guides in our official documentation).

If you want to share additional guides, please do so, in our Community Guides wiki on the community forum: <https://community.home-assistant.io/c/community-guides/51>

Thanks! 👍